ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -102,12 +102,80 @@ function filter_test_by_fips() {
     fi
     echo_e2e_tags
 }
+function filter_test_by_capability() {
+    local enabledcaps xversion yversion
+    enabledcaps="$(oc get clusterversion version -o yaml | yq '.status.capabilities.enabledCapabilities[]')"
+    IFS='.' read xversion yversion _ < <(oc version -o yaml | yq '.openshiftVersion')
+    local v411 v412 v413 v414 v415 v416
+    v411="baremetal marketplace openshift-samples"
+    v412="${v411} Console Insights Storage CSISnapshot"
+    v413="${v412} NodeTuning"
+    v414="${v413} MachineAPI Build DeploymentConfig ImageRegistry"
+    v415="${v414} OperatorLifecycleManager CloudCredential"
+    v416="${v415}"
+    # [console]=console
+    # the first `console` is the capability name
+    # the second `console` is the tag name in verification-tests
+    declare -A tagmaps
+    tagmaps=([baremetal]=xxx
```

Comment:
Do we really need xxx for all tags? I think this map only contains existing tags is enough.

Reply:
Those `xxx` are placeholders. We will have to replace them with real tags after communicating with related feature owners.

Reply:
So shall we only add tags after getting confirmation from their owners? Then at below if statement we can only check if the tag exists.

Reply:
Actually I explicitly set them like this, so that
1. Only need to compare this list with another several lists (v411 till v416) for one time, rather than every one I went back to this.
2. Those `xxx` one adds a TODO message in the log, I can easily find out which one is done, and which one is not rather than just silently ignore those TODOs. We might forgot them weeks/months later as we can't got those info timely if we just ignore them silently.

Reply:
OK, it makes sense.